contracts/interchain-token-service/src/abi.rs

```diff
@@ -478,6 +535,7 @@ mod tests {
                     original
                         .clone()
                         .abi_encode(&env)
+                        .unwrap()
```

Comment:
same everywhere. `.unwrap` can be used when dropping error for some helper call we know will succeed